### PR TITLE
docs: update password translation in malay

### PR DESCRIPTION
**Pull request by kamaroolkarim**

### CODE REVIEW

File: src/ms/validation.php [hunk 1]
```diff
@@ -92,7 +92,7 @@
     'not_in'               => ':attribute tidak sah.',
     'not_regex'            => 'Format :attribute adalah tidak sah.',
     'numeric'              => ':attribute mesti nombor.',
-    'password'             => 'The password is incorrect.',
+    'password'             => ':attribute tidak sah.',
```
Comment (caouecs) [inline src/ms/validation.php:95]: Why do you add :attribute ?

**Review by caouecs — CHANGES_REQUESTED** [review]
Thank you, but a typo

File: src/ms/validation.php (hunk 1 above)
Comment (kamaroolkarim) [reply src/ms/validation.php:95]: so no need attribute? let me change to 'Kata laluan tidak sah' then

Comment (andrey-helldar) [reply src/ms/validation.php]: The original text of this line indicates that the password is incorrect. It has no reference to the attribute name.

**Review by andrey-helldar — APPROVED** [review]

File: src/ms/validation.php (hunk 1 above)
Comment (kamaroolkarim) [reply src/ms/validation.php]: @andrey-helldar ok noted. already fixed it

**Comment by caouecs** [comment]
Thank you

**Pull request merged**